crates/plugin/src/sender.rs

```diff
@@ -103,12 +103,9 @@ impl Sender {
         }
 
         metrics.reconnects.log(1);
-        let prod = match self.connect(prod).await.map_err(log_err(&metrics.errs)) {
-            Ok(p) => p,
-            Err(()) => return,
-        };
+        let Ok(p) = self.connect(prod).await.map_err(log_err(&metrics.errs)) else { return };
```

Comment:
Interesting syntax :eyes:

Reply:
yeah it's new to this newer version of rust.  `else` statements in `let` pattern match deconstruction